recipe/meta.yaml

```diff
@@ -60,8 +61,8 @@ about:
       traditional STL
     - Provide a non-proprietary, plain text model format that can be edited and
       executed with only a web browser
-  doc_url: http://dcowden.github.io/cadquery/
-  dev_url: https://github.com/dcowden/cadquery
+  doc_url: https://cadquery.readthedocs.io/
+  dev_url: https://github.com/cadquery/cadquery
 
 extra:
   recipe-maintainers:
```

Comment:
please feel free to add yourself as a maintainer: i clearly haven't been keeping abreast of the releases!